modules/installation-creating-azure-control-plane.adoc

```diff
@@ -36,7 +36,7 @@ This template describes the control plane machines that your cluster requires.
 +
 [source,terminal]
 ----
-$ export MASTER_IGNITION=`cat <installation_directory>/master.ign | base64`
+$ export MASTER_IGNITION=`cat <installation_directory>/master.ign | base64 -w0`
```

Comment:
I tested it with this command, however I notice we use `base64 | tr -d '\n'` in the [upstream docs](https://github.com/openshift/installer/blob/master/docs/user/azure/install_upi.md#launch-the-permanent-control-plane)

Reply:
I noticed that too. From what I gathered, `base64 | tr -d '\n'` was compatible for Linux **AND** macOS. I'm fine with matching the upstream docs if that makes sense.

Reply:
You're right. We should use`base64 | tr -d '\n'` instead of  `base64 -w0` for compability (https://github.com/openshift/installer/pull/4090#discussion_r476597902)